javascript/javascript2/week3/log_location.js

```diff
@@ -0,0 +1,32 @@
+//Warm up Five (Create a button with the text called "Log location")
+function geoFindMe() {
+
+    const status = document.querySelector('#status');
+    const mapLink = document.querySelector('#map-link');
+
+    mapLink.href = '';
+    mapLink.textContent = '';
+
+    function success(position) {
```

Comment:
try finding better name
```suggestion
    function printLocation(position) {
```